scrapers/AlexLegend/AlexLegend.yml

```diff
@@ -0,0 +1,48 @@
+name: AlexLegend
+
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - alexlegend.com/video/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[contains(concat(" ", normalize-space(@class), " "), " content-title ")]/h1
+      Code:
+        selector: //link[@rel="canonical"]/@href
+        postProcess:
+          - replace:
+            - regex: .+-(\d+)\..+
+              with: $1
+      Details: //div[contains(concat(" ", normalize-space(@class), " "), " content-desc ")][2]

```

Comment:
description is also available in meta when not available in the page

[example](https://alexlegend.com/video/skylar-snow-black-lingerie-surprise-sex-behind-the-scenes-1074.html)

Reply:
Meta description is the page description and not related to the scene.
Meta description template: 'Watch {SCENE_NAME} on AlexLegend, the best HD...'